src/qibolab/result.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from scipy import signal
+
+
+class ExecutionResult:
```

Comment:
Yet another `dataclass` candidate. But apart from using types for `__init__`, this is already pretty good (yes, simplicity has a big impact on goodness in my mind...).